components/AddButton.js

```diff
@@ -0,0 +1,25 @@
+import AddIcon from '../public/assets/add-icon.svg';
+import styled from 'styled-components';
+
+export default function AddButton({ setAddActive }) {
+  return (
+    <>
+      <StyledButton
+        onClick={() => {
+          setAddActive(true);
+          console.log(true);
+        }}
```

Comment:
```suggestion
        onClick={onClick}
```

I would pass the information as an onClick until you reach the component where you actually wan to work with the onClick event :)

Also: always delete your console logs before commiting :)